AllReadyApp/Web-App/AllReady/Views/Campaign/ManageCampaign.cshtml

```diff
@@ -0,0 +1,102 @@
+﻿@model IList<AllReady.ViewModels.Campaign.CampaignViewModel>
+@using AllReady.Security
+@using AllReady.ViewModels.Campaign
+
+@{
+    ViewData["Title"] = "All Registered Campaigns";
+    @addTagHelper "*, Microsoft.AspNetCore.Mvc.TagHelpers"
```

Comment:
this should be removed b/c C:\Projects\allReady\AllReadyApp\Web-App\AllReady\Views\_ViewImports.cshtml already contains an `@addTagHelper` directive.